main.go

```diff
@@ -53,6 +56,8 @@ func init() {
 	kingpin.Flag("labels", "A set of labels to restrict the list of affected pods. Defaults to everything.").StringVar(&labelString)
 	kingpin.Flag("annotations", "A set of annotations to restrict the list of affected pods. Defaults to everything.").StringVar(&annString)
 	kingpin.Flag("namespaces", "A set of namespaces to restrict the list of affected pods. Defaults to everything.").StringVar(&nsString)
+	kingpin.Flag("include-pattern", "Regex pattern for pod names to include.  All included by default").StringVar(&includePattern)
+	kingpin.Flag("exclude-pattern", "Regex pattern for pod names to exclude.  None excluded by default").StringVar(&excludePattern)
```

Comment:
Let's use `RegexpVar` instead of `StringVar`. With that `kingpin` takes care of parsing the flag, converting it into a RegExp object as well `fataling` when it's an invalid regexp.

https://github.com/alecthomas/kingpin/blob/a39589180ebd6bbf43076e514b55f20a95d43086/values_generated.go#L720-L722